src/status_im/data_store/realm/schemas/base/v4/account.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.data-store.realm.schemas.base.v4.account
+  (:require [taoensso.timbre :as log]
+            [status-im.constants :as constants]))
+
+(def schema {:name       :account
+             :primaryKey :address
+             :properties {:address             :string
+                          :public-key          :string
+                          :updates-public-key  {:type     :string
+                                                :optional true}
+                          :updates-private-key {:type     :string
+                                                :optional true}
+                          :name                {:type :string :optional true}
+                          :phone               {:type :string :optional true}
+                          :email               {:type :string :optional true}
+                          :status              {:type :string :optional true}
+                          :debug?              {:type :bool :default false}
+                          :photo-path          :string
+                          :signing-phrase      {:type :string}
+                          :last-updated        {:type :int :default 0}
+                          :signed-up?          {:type    :bool
+                                                :default false}
+                          :network             :string
+                          :networks            {:type       :list
+                                                :objectType :network}}})
+
+(defn migration [_old-realm new-realm]
+  (log/debug "migrating account schema v4")
+  (let [accounts (.objects new-realm "account")]
+    (dotimes [i (.-length accounts)]
+      (let [account (aget accounts i)]
+        (aset account "network" constants/default-network)))))
+        ;; creation of network object crashes the app
+        ;(aset account "networks" (clj->js (vals constants/default-networks)))))))
```

Comment:
What's this comment? Don't understand

Reply:
@flexsurfer please elaborate

Reply:
originally i implemented this using this commented line, but after updating realm, app started crashes, we had to find a workaround, i decided not to remove this, maybe for the future

Reply:
I think it's quite confusing, because now, when `constants/default-networks` already contains js object values, uncommenting the `clj->js` would crash it for sure.
I'm in favor of removing the line altogether, maybe adding `TODO` comment in the constants namespace.